lib/git_repository/cloning.rb

```diff
@@ -29,25 +29,25 @@ def clone_svn(url)
         fetch:    ':refs/remotes/git-svn'
     end
 
-    set_section %w( svn-remote svn ), options
+    set_section %w(svn-remote svn), options
     pull_svn
   end
 
   def pull
     with_head_change head_oid do
-      git_exec 'remote', 'update'
+      git_exec('remote', 'update')
     end
   end
 
   # Fetches the latest commits and resets the local master
   def pull_svn
     old_head_oid = head_oid
-    git_exec 'svn', 'fetch'
+    git_exec('svn', 'fetch')
 
     if svn_has_trunk?
-      reset_branch old_head_oid, 'master', "remotes/trunk"
+      reset_branch(old_head_oid, 'master', "remotes/trunk")
     else
-      reset_branch old_head_oid, 'master', "remotes/git-svn"
+      reset_branch(old_head_oid, 'master', "remotes/git-svn")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.